static/javascripts/chat.js

```diff
@@ -1,5 +1,38 @@
 // Chat adapted from: https://github.com/socketio/socket.io/tree/master/examples/chat
 var chatRoom;
+var numChatEvent = 0;
+var curentlyfocused = 1;
+
+var title = document.title;
+
+function newUpdate() {
+    update = setInterval(changeTitle, 2000);
+}
+
+// Set up event handler for the window focus event
+window.addEventListener("focusin", function(event) 
+{ 
+    curentlyfocused = 1;
+    numChatEvent = 0;
+    document.title = title;
+}, false);
+
+// Set up event handler for the window blur event
+window.addEventListener("focusout", function(event) 
+{ 
```

Comment:
It would be nice to have the same convention we use for breaks in function definition.
Also we seem to always use Jquery for adding the event listener.